### ISSUE TITLE
Activities should be sorted by name

### ISSUE THREAD

**Issue by finnhodgkin** [TwinePlatform/DataPower#171]
At the moment if one of the activities is changed, the display order changes.

It's a minor thing but if there are repeated changes to an activity between visits it's hard to follow what's going on. Also if a community business has a whole bunch of activities then the list should be ordered for clarity.

**Comment by eliascodes** [comment]
Think this is fixed with the new component #330
